src/pymor/algorithms/rules.py

```diff
@@ -267,11 +268,11 @@ def apply(self, obj):
         For each |rule|, it is checked if it :meth:`~rule.matches` the given
         object. If `False`, the next |rule| in the table is considered.
         If `True` the corresponding :attr:`~rule.action` is executed with
-        `obj` as parameter. If execution of :attr:`~action` raises
+        `obj` as parameter. If execution of :attr:`~rule.action` raises
```

Comment:
Huh, why can this attribute be linked whereas so many other cannot?

Reply:
`rule` has an "Attibutes" section, so that might be it.